app/src/main/java/org/sagebionetworks/research/mpower/medication/MedicationActivity.kt

```diff
@@ -0,0 +1,109 @@
+package org.sagebionetworks.research.mpower.medication
+
+
+import android.arch.lifecycle.Observer
+import android.arch.lifecycle.ViewModelProviders
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.support.v7.widget.LinearLayoutManager
+import android.text.SpannableString
+import android.text.style.UnderlineSpan
+import kotlinx.android.synthetic.main.activity_medication.*
+import org.slf4j.LoggerFactory
+
+import org.sagebionetworks.research.mpower.R
+import org.sagebionetworks.research.mpower.TaskLauncher
+import javax.inject.Inject
+
+
+class MedicationActivity : AppCompatActivity(),
```

Comment:
@liujoshua Do you think this should be a separate activity, or show we try to use PerformTaskActivity and have the medication task be a separate step like the other tracking tasks?

Reply:
I can change to whatever is appropriate.  I just started with an activity cause I was sure where to hook in.

Reply:
Let's merge in as is and switch to Fragment as a refactor